### ISSUE TITLE
Raster converted to data frame missing output for 'forest type' column

### ISSUE THREAD

**Issue by ngould7** [rspatial/terra#441]
I'm using the terra package to convert multiple rasters into columns in a data frame.  One raster, 'forest type' (zip file attached with .tif file) will not populate with the 'class names' for each forest type/pixel instead of the numeric values (associated with each forest type) for each pixel.  The forest type column that should list each forest type class name (e.g., white oak, white pine, etc.) has no populated information. 

I've included a simple r script that duplicates the error.  

[Forest_Type_Raster_(tiff)_R_script.zip](https://github.com/rspatial/terra/files/7693744/Forest_Type_Raster_.tiff._R_script.zip)

**Comment by rhijmans** [comment]
Thank you very much for reporting this. I now get:


```
library(terra)
#terra version 1.5.1
r <- rast("For_Type_100.tif")
x <- crop(r, ext(1473228, 1475029, -50963, -49462) )

d <- as.data.frame(x, na.rm = FALSE, cells = TRUE, xy = TRUE)
head(d)
#  cell       x         y                       CLASS_NAME
#1    1 1473279 -49512.36 Yellow-poplar/White Oak/Northern
#2    2 1473379 -49512.36 Yellow-poplar/White Oak/Northern
#3    3 1473479 -49512.36               Shortleaf Pine/Oak
#4    4 1473579 -49512.36               Shortleaf Pine/Oak
#5    5 1473679 -49512.36               Shortleaf Pine/Oak
#6    6 1473779 -49512.36 Yellow-poplar/White Oak/Northern
 
freq(x)
#  layer                            value count
#1     1                                     30
#2     1                    Loblolly Pine     9
#3     1                   Shortleaf Pine     6
#4     1               Shortleaf Pine/Oak    41
#5     1        White Oak/Red Oak/Hickory     4
#6     1 Yellow-poplar/White Oak/Northern    77
#7     1           Sweetgum/Yellow-poplar   103
```

**Comment by ngould7** [comment]
Thank you very Robert!  That was incredibly fast too.  However, my issue is not fixed, do I need to update to version 1.5.1?
________________________________

**Comment by rhijmans** [comment]
That is correct. If you are on windows, the easiest way is to run 

`install.packages('terra', repos='https://rspatial.r-universe.dev')` 

(a couple of hours after the update was made; otherwise it might still be an earlier version)
